### ISSUE TITLE
[IMP] Add Logo to Menu Icon

### ISSUE THREAD

**Issue by lasley** [clouder-community/clouder#181]
Currently there is no web icon for Clouder, boooo!

![image](https://cloud.githubusercontent.com/assets/7397578/21172837/2e027932-c189-11e6-81f4-9d1acb2aa178.png)

It's a simple add, see [this PR](https://github.com/OCA/vertical-medical/pull/126/files). I would do it, but I'm not sure what type of branding you have available.

If you put it in `static/description/icon.png` it will also double as the Icon for the Apps store / section.

Another thing just as kind of a side note - Only `clouder` should be marked as an `application` in the manifest file. That's basically the indicator of whether the app gets a menu icon. Marking all the templates as so kind of pollutes the apps namespace.

What would be nice though would be a `base_config_settings` to enable templates, similar to the other random modules that get installed by that. Maybe I should make another issue for that now that I'm thinking about it.

**Comment by nicholas-itpedia** [comment]
I can work on the web icon.
@YannickB @lasley do you have any ideas or prior examples?

**Comment by lasley** [comment]
@nicholas-itpedia - I'm pretty bad at this type of thing, so definitely take this for a grain of salt. Maybe two clouds with a plug between them or something? @YannickB?

**Comment by nicholas-itpedia** [comment]
@lasley we have been working on a few examples:

![image](https://cloud.githubusercontent.com/assets/12131726/23282337/b34a8aa6-f9ee-11e6-8731-7a8f2acf54c7.png)

**Comment by nicholas-itpedia** [comment]
Example 2

![image](https://cloud.githubusercontent.com/assets/12131726/23282362/cfe7e5d2-f9ee-11e6-99fb-4c658ecb8074.png)

**Comment by nicholas-itpedia** [comment]
Example 3

![image](https://cloud.githubusercontent.com/assets/12131726/23282406/01d704f6-f9ef-11e6-8060-2c147b7b40ff.png)

**Comment by YannickB** [comment]
Yes, I suggested the shaking hands at first, but looking twice I find it a bit formal. Here was my last answer to it : 

I'm wondering if the two shaking hands is really that much a good idea. This may be a bit too much formal, and while I want insist on the business side, collaboration is the most important point.
Maybe several hands joigning together is better like in https://www.cdc.gov/foodsafety/images/flexslider/group-with-hands-together-large.jpg or http://tweakyourbiz.com/growth/files/How-Collaboration-Creates-Growth-For-Your-SME.jpg, while still keeping the same color/style/hands going out from some clouds. 

I understand this may be difficult to do though, and I'm not completely sure of the idea.

**Comment by nicholas-itpedia** [comment]
Version 4

![image](https://cloud.githubusercontent.com/assets/12131726/23313282/14d6c52c-fa8b-11e6-94ad-d997c5e1dcd5.png)

**Comment by nicholas-itpedia** [comment]
Version 5

![image](https://cloud.githubusercontent.com/assets/12131726/23313292/213138d4-fa8b-11e6-9141-89337e700d9d.png)

**Comment by nicholas-itpedia** [comment]
Version 6

![image](https://cloud.githubusercontent.com/assets/12131726/23313304/2a6084c8-fa8b-11e6-8ada-fd03c061e8ec.png)

**Comment by nicholas-itpedia** [comment]
Version 7

![image](https://cloud.githubusercontent.com/assets/12131726/23313320/3df75ed0-fa8b-11e6-9d74-ae19659503fb.png)

**Comment by nicholas-itpedia** [comment]
Version 8

![image](https://cloud.githubusercontent.com/assets/12131726/23313332/47c1f862-fa8b-11e6-9073-158cb85d907d.png)
